tests/Functional/AnnotationsTest.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+/**
+ * @package   Functional-php
+ * @author    Hugo Sales <hugo@fc.up.pt>
+ * @copyright 2020 Hugo Sales
+ * @license   https://opensource.org/licenses/MIT MIT
+ * @link      https://github.com/lstrojny/functional-php
+ */
+
+namespace Functional\Tests;
+
+use ReflectionFunction;
+
+use function Functional\group;
+use function get_defined_functions;
```

Comment:
Do we or do we not import the root namespaced functions? It looks like a mix at the moment.

Reply:
It’s a mess, yeah. Functional shows it's age here (I just realized when updating the copyright notice it’s 10 years anniversary). When it started, `use function …` was not a thing. But yes, it should be imported everywhere.